src/main/resources/i18n/challenges/injection/ac8f3f6224b1ea3fb8a0f017aadd0d84013ea2c80e232c980e54dd753700123e.properties

```diff
@@ -1,5 +1,5 @@
 challenge.challengeName = XXE Injection 1
-challenge.description =  To complete this challenge you must read the contents JSON's file 
+challenge.description =  To complete this challenge you must read the contents of file 
```

Comment:
Add `the` between `of file`

like so

`To complete this challenge you must read the contents of the file`